packages/fast-components-react-base/src/button/__snapshots__/button.spec.ts.snap

```diff
@@ -3,6 +3,7 @@
 exports[`button button: 0 1`] = `
 <button
   className="button"
+  href={null}
```

Comment:
This feels weird to me. href should never be written to the button DOM, yet our snapshot shows it. Maybe it would be better to return href as an attribute only when the tag is an anchor

Reply:
I'm fine with that but we should treat all attributes the same (IE, checkbox disabled - disabled={this.props.disabled || null})

Reply:
And as a note, while it's in the snapshot, it's not in the DOM.

Reply:
It does look weird - I think that is just a product of snapshots though - react handles it correctly